docs/developer-guide/debugging.md

```diff
@@ -0,0 +1,50 @@
+# Debugging
+
+Debugging in prowler make things easier!
+Here is an example of the debugging configuration file that you can add to vscode to debug prowler:
```

Comment:
- Where should I put that file to let VSCode know that this is my configuration?
- What should be the name of that file?

Reply:
Done